jdk-1.7-parent/annotation/src/main/java/org/wicketstuff/annotation/scan/AnnotatedMountScanner.java

```diff
@@ -97,46 +98,16 @@
 {
 
 	/**
-	 * Get the Spring search pattern given a package name or part of a package name
-	 * 
-	 * @param packageName
-	 *            a package name
-	 * @return a Spring search pattern for the given package
-	 */
-	public String getPatternForPackage(String packageName)
-	{
-		if (packageName == null)
-			packageName = "";
-		packageName = packageName.replace('.', '/');
-		if (!packageName.endsWith("/"))
-		{
-			packageName += '/';
-		}
-
-		return "classpath*:" + packageName + "**/*.class";
-	}
-
-
-	/**
-	 * Scan given a package name or part of a package name and return list of classes with MountPath
-	 * annotation.
-	 * 
-	 * @return A List of classes annotated with &#64;MountPath
-	 */
-	public List<Class<?>> getPackageMatches(String pattern)
```

Comment:
Same here.
I think it'd be better to keep these public methods implemented with Reflections.